x/capability/genesis.go

```diff
@@ -13,11 +13,12 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 		panic(err)
 	}
 
-	// set owners for each index and initialize capability
+	// set owners for each index
 	for _, genOwner := range genState.Owners {
 		k.SetOwners(ctx, genOwner.Index, genOwner.IndexOwners)
-		k.InitializeCapability(ctx, genOwner.Index, genOwner.IndexOwners)
 	}
+	// initialize in-memory capabilities
+	k.InitMemStore(ctx)
 }
```

Comment:
This will be used if node is restarted from genesis (either a new chain or genesis restart)